integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -30,6 +30,15 @@ properties:
   # alluxio.user.metrics.collection.enabled: 'true'
   alluxio.security.stale.channel.purge.interval: 365d
 
+tieredstore:
```

Comment:
@cheyang this is redundant. scroll below https://github.com/Alluxio/alluxio/commit/3added565773609aabc71eccaa5cdeefff55c164#diff-c26bfed7e349bf27fd0554872241278eL124

The default config is already specified as ```# Tiered Storage
tieredstore:
  levels:
  - alias: MEM # Medium type
    level: 0
    path: /dev/shm
    type: emptyDir
    quota: 1G
    high: 0.95
    low: 0.7
```

Reply:
Yes, you're right.